examples/step-49/doc/results.dox

```diff
@@ -271,7 +271,7 @@ Triangulation<3> triangulation;
 create_3d_grid(triangulation);
 triangulation.set_manifold(cylinder_id, cylinder);
 
-for (auto cell : triangulation.active_cell_iterators())
+for (auto & cell : triangulation.active_cell_iterators())
```

Comment:
Is there a reason why clang-format sometimes puts a space between `& cell` and sometimes not?

Reply:
This is in a .dox file. clang-format doesn't touch those. I suspect that @tamiko used the version with the space in his regex-replace, and then clang-format removed the space in all places it gets to see.

Reply:
Exactly